Gemfile

```diff
@@ -103,4 +103,6 @@ end
 
 gem 'contracts', '< 0.16' if RUBY_VERSION < '1.9.0'
 
+gem 'drb' if RUBY_VERSION >= '3.4'
```

Comment:
If this is fixing [or attempting to] fix an extraction of `drb` on Ruby head it is not enough, we rely on drb and it will cause weird issues if its missing, so we need to actually detect the presence of drb [and test for it] issuing a warning etc.

Reply:
Fair enough. I’ll implement a warning separately.
We do rely on drb in bisect, don’t we? But we have a fallback?

Reply:
In the bisector but also there is a DRb mode that supported things like spork and other things, we will need to check for any usage and appropriately load it / issue a warning when it is not present